docker-compose.yml

```
@@ -0,0 +1,41 @@
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - "3000:3000" 
+    networks:
+      - app-network
+
+  caddy:
+    image: caddy:latest
+    ports:
+      - "80:80"    
```

yamllint

[error] trailing spaces

(trailing-spaces)